### ISSUE TITLE
The builder’s visual highlighting not working

### ISSUE THREAD

**Issue by Nevrai** [McJtyMods/RFToolsBase#83]
The RFTools builder does not show the area that it’s working on with the visual highlighting option enabled. I disabled my shaders to see if that caused it, but it didn’t work.

For the record, [Rubidium](https://www.curseforge.com/minecraft/mc-mods/rubidium) and [Oculus](https://www.curseforge.com/minecraft/mc-mods/oculus) are installed, but I’ve had no issues with visual highlighting in other mods. Not sure if they might have something to do with this.

I’m using a clearing Fortune quarry card in the builder. As you can see in the GIF below, there’s no highlighting around the area it’s quarrying. It doesn’t highlight the area before starting the builder either.

![RFTools builder bug (low fps version)](https://user-images.githubusercontent.com/59761487/187019193-aa590e70-c485-4c11-915f-2aa9c6600d41.gif)

### Versions
* **Modpack:** [All the Mods 7](https://www.curseforge.com/minecraft/modpacks/all-the-mods-7) (0.4.29—latest)
* **Minecraft:** 1.18.2
* **Forge:** 40.1.68
* **RFToolsBase:** 3.0.10 (latest for 1.18.2)
* **RFToolsBuilder:** 40.0.14 (latest for 1.18.2)

**Comment by McJty** [comment]
You have highlighting enabled in the builder gui?

Also might be interesting to try without those visual mods

**Comment by Nevrai** [comment]
Yes, I have highlighting enabled in the builder. I can try it later without Rubidium and Oculus installed and see if it works then.

**Comment by dev-iain** [comment]
has there been any update on this? this is still an issue
